### ISSUE TITLE
closures.write_closure_info_to_log(seismic.rfc_raw[1:4, ...])

### ISSUE THREAD

**Issue by klaas-koster** [sede-open/synthoseis#22]
The call in main.py to write_closure_info_to_log() is hardwired to pass on the second, third, and fourth angle. That looks like a remnant from the past where the code assumed the user always specified three angles in the configuration file and the code padded those with a 0- and 45-degree response. I suggest changing

closures.write_closure_info_to_log(seismic.rfc_raw[1:4, ...])

to

closures.write_closure_info_to_log(seismic.rfc_raw[:, ...])

Note, that this is not guaranteed to give good results if the user specifies just one angle in the configuration file.

**Comment by gavargas22** [comment]
Thank you for all your great analysis of the repo. These are great ideas

**Comment by geokefe** [comment]
I'm having problem with the installation; how do we install synthoseis in Jupiter notebook?

**Comment by klaas-koster** [comment]
@geokefe this is not the right place to ask that question. 
Anyway, there is a 'notebook' folder that show exactly how to use synthoseis in a Jupyter notebook.

**Comment by gavargas22** [comment]
> I'm having problem with the installation; how do we install synthoseis in Jupiter notebook?

Indeed, you can submit a new issue for any question you have so we can keep track of it in a separate ticket.

I will make a new issue to discuss an answer to your question

**Comment by gavargas22** [comment]
Closing this as resolved on `master`. The hardwired slice was replaced with a dynamic one that accounts for both the configured incident angles and the QC padding volumes:

```python
n_incident = len(p.incident_angles)
rfc_start = 1 if p.model_qc_volumes else 0
closures.write_closure_info_to_log(seismic.rfc_raw[rfc_start : rfc_start + n_incident, ...])
```

This also explains the "sometimes 3, sometimes 5 angles in `rfc_raw`" observation: when `model_qc_volumes` is enabled, 0° and 45° QC volumes are prepended/appended around the configured angles, so `rfc_raw` carries `n_incident + 2` volumes. The slice above now selects exactly the configured angles in either case.

@geokefe regarding the installation question in this thread — as noted, the `notebooks/` folder shows how to use synthoseis from a Jupyter notebook. Please open a separate issue if you still run into installation problems.
